README.md

```diff
@@ -183,6 +183,25 @@ kitty target window
     This is the id of the kitty window that you wish to target.
     See e.g. the value of $KITTY_WINDOW_ID in the target window.
 
+kitty listen on
+
+    Specifies where kitty should listen to control messages.
+    See e.g. the value of $KITTY_LISTEN_ON in the target window.
+
+To work properly, `kitty` must also be started with the following options:
+
+```sh
+kitty -o allow_remote_control=yes -o enabled_layouts=tall --listen-on unix:/tmp/mykitty
```

Comment:
is `-o enabled_layouts=tall` necessary?
(same below @ line 201)

Reply:
It is very likely not needed here